### PR TITLE
Added Six speakers

### LINKED ISSUE #265: Add speaker Tiejun Chen
tiejunchen

Please use the above photo and get his details from https://eventyay.com/e/b548bcfc/#speakers

### PR DESCRIPTION
**Pull request by girijamanojkumarreddy**
## Fixes
#265
#287
#288
#290
#295
#296

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3098,7 +3098,113 @@ <h1>Speakers</h1>
...
                         </div>
-                
+
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                                <div class="image-holder">
+                                    <img class="background-image" alt="Hemant Dangi" src="img/Hemant_Dangi.jpg">
+                                    <div class="hover-state text-center preserve3d">
+                                        <div class="social-links vertical-align">
+                                            <a target="default" href="https://www.linkedin.com/in/hemantdangi/"><i class="icon social_linkedin"></i></a>
```
Comment (gautamjajoo) [inline index.html]: Please add FaceBook and the External link also.

File: index.html [hunk 2]
```diff
@@ -3098,7 +3098,113 @@ <h1>Speakers</h1>
...
+                                    <img class="background-image" alt="Rakesh Tadisetty" src="img/Rakesh_Tadishetty.jpg">
+                                    <div class="hover-state text-center preserve3d">
+                                        <div class="social-links vertical-align">
+                                            <a target="default" href="http://rakesh.studentzeal.com/"><i class="icon fa fa-external-link"></i></a>
+                                            <a target="default" href="https://twitter.com/TRakeshSREC"><i class="icon social_twitter"></i></a>
+
+                                        </div>
+                                    </div>
+                                </div>
+                                <span class="speaker-name">Rakesh Tadisetty</span>
```
Comment (gautamjajoo) [inline index.html]: ** Tadishetty

File: index.html [hunk 3]
```diff
@@ -3098,7 +3098,113 @@ <h1>Speakers</h1>
...
+                            <div class="speaker">
+                                <div class="image-holder">
+                                    <img class="background-image" alt="Divyam madaan" src="img/Divyammadaan.jpg">
+                                    <div class="hover-state text-center preserve3d">
+                                        <div class="social-links vertical-align">
+                                            <a target="default" href="https://divyam3897.github.io/"><i class="icon fa fa-external-link"></i></a>
+                                            <a target="default" href="https://twitter.com/divyam3897"><i class="icon social_twitter"></i></a>
+
+                                            <a target="default" href="https://github.com/divyam3897"><i class="icon fa fa-github"></i></a>
+                                        </div>
```
Comment (gautamjajoo) [inline index.html]: Please add the Linkedin link also.

File: index.html [hunk 4]
```diff
@@ -3098,7 +3098,113 @@ <h1>Speakers</h1>
...
+                                    </div>
+                                </div>
+                                <span class="speaker-name">Divyam madaan</span>
```
Comment (gautamjajoo) [inline index.html]: **Madaan

File: index.html [hunk 5]
```diff
@@ -3098,7 +3098,113 @@ <h1>Speakers</h1>
...
+                            <div class="speaker">
+                                <div class="image-holder">
+                                    <img class="background-image" alt="Salman Rahman Desh" src="img/Salman_Rahman_Desh.jpg">
+                                    <div class="hover-state text-center preserve3d">
+                                        <div class="social-links vertical-align">
+                                            <a target="default" href="http://srdesh.info/"><i class="icon fa fa-external-link"></i></a>
+                                            <a target="default" href="https://twitter.com/Salman_Desh"><i class="icon social_twitter"></i></a>
+                                            <a target="default" href="https://www.linkedin.com/in/salmanrahmandesh/"><i class="icon social_linkedin"></i></a>
+                                            <a target="default" href="https://github.com/srdesh"><i class="icon fa fa-github"></i></a>
+                                        </div>
```
Comment (gautamjajoo) [inline index.html]: Please add Facebook link also.

Comment (girijamanojkumarreddy) [reply index.html]: Thank you for the suggestions I will implement them now

**Comment by girijamanojkumarreddy** [comment]
Sorry for the delay and the Preview link is [here](http://bit.ly/2Foysxg)

**Comment by mariobehling** [comment]
Thanks, but Hemant Dangi is too small. I will merge that anyways and it should be fixed afterwards.

**Pull request merged**